### PR TITLE
MAGEDOC-3963-price-rule-custom-options

### PR DESCRIPTION
**Pull request by ccondra**
image

### CODE REVIEW

**Comment by ccondra** [comment]
Running tests

**Comment by ccondra** [comment]
Running tests

File: src/catalog/settings-advanced-custom-options.md [hunk 1]
```diff
@@ -65,126 +75,11 @@ Adding customizable options to a product is an easy way to offer customers a sel
...
-               </table>
-            </td>
-         </tr>
-      </tbody>
-   </table>
+## Input Types ##
+
+|Type|Description|
+|--- |--- |
+|**Text**|An input line or text box where the customer can enter the required information. Options:<br />**Field** - A  single line input field for text.<br />**Area** - A multiple-line input field. Text can be formatted with the WYSIWYG editor, or typed as HTML directly into the text box.|
```
Comment (leslietilling) [inline src/catalog/settings-advanced-custom-options.md]: The items in the Type column should not be bold

File: src/catalog/settings-advanced-custom-options.md [hunk 2]
```diff
@@ -65,126 +75,11 @@ Adding customizable options to a product is an easy way to offer customers a sel
...
+|**File**|Allows the customer to upload a file.|
+|**Select**|Allows the customer to select a single option, or multiple options, depending on input type used. Options:<br />**Drop-down** - A drop-down list of options that allows only one selection.<br />**Radio Buttons** - A set of options that allows only one selection.<br /> **Checkbox** - A checkbox is a variation of a yes/no option. If the product has more than one checkbox, multiple selections can be made.<br />**Multiple Select** - A drop-down list box of options that accepts multiple selections. To choose multiple options, hold down the Ctrl (PC) or Command (Mac) key and click each option.|
```
Comment (leslietilling) [inline src/catalog/settings-advanced-custom-options.md]: Just a fix for extra space

```suggestion
|**Select**|Allows the customer to select a single option, or multiple options, depending on input type used. Options:<br />**Drop-down** - A drop-down list of options that allows only one selection.<br />**Radio Buttons** - A set of options that allows only one selection.<br />**Checkbox** - A checkbox is a variation of a yes/no option. If the product has more than one checkbox, multiple selections can be made.<br />**Multiple Select** - A drop-down list box of options that accepts multiple selections. To choose multiple options, hold down the Ctrl (PC) or Command (Mac) key and click each option.|
```

File: src/marketing/price-rules-cart.md [hunk 3]
```diff
@@ -7,16 +7,16 @@ Cart price rules apply discounts to items in the shopping cart, based on a set o
...
 
-<!--{% if "Default.CE Only" contains site.edition %}-->
-![Cart price rule]({% link images/images/price-rule-cart.png %}){: .zoom}
-_Cart Price Rules_
-<!--{% endif %}-->
-<!--{% if "Default.EE-B2B" contains site.edition %}-->
-![Cart price rule]({% link images/images/price-rules-cart-ee.png %}){: .zoom}
-_Cart Price Rules_
-<!--{% endif %}-->
+## To access Cart Price Rules:
```
Comment (leslietilling) [inline src/marketing/price-rules-cart.md]:
```suggestion
## Access Cart Price Rules
```

File: src/catalog/settings-advanced-custom-options.md (hunk 1 above)
Comment (ccondra) [reply src/catalog/settings-advanced-custom-options.md]: OK, I'll fix that. thanks!

File: src/marketing/price-rules-cart.md (hunk 3 above)
Comment (ccondra) [reply src/marketing/price-rules-cart.md]: All of the instructions begin with"To do this or that."   It's not a heading. Am I using the wrong style?

**Comment by ccondra** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Comment by ccondra** [comment]
running tests

**Pull request merged**